src/Reflection/SignatureMap/functionMap.php

```diff
@@ -1602,6 +1602,7 @@
 'DateTime::__wakeup' => ['void'],
 'DateTime::add' => ['static', 'interval'=>'DateInterval'],
 'DateTime::createFromFormat' => ['static|false', 'format'=>'string', 'time'=>'string', 'timezone='=>'DateTimeZone|null'],
+'DateTime::createFromImmutable' => ['static|null', 'object'=>'DateTimeImmutable'],
```

Comment:
When it returns null?

Reply:
Here an example: https://3v4l.org/oQnFA

I found this case by looking at the following test:
https://github.com/php/php-src/blob/852485d8ecd784153e41e565a0a87abf99cf4e0d/ext/date/tests/DateTime_createFromImmutable.phpt

Reply:
I think we can omit this case. It would be too annoying for the correct usage for levels 6+. `DateTimeImmutable::createFromMutable` also does not have nullable typehint.

Reply:
Ok, done :)